src/askQuestion.js

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable no-console */
+'use strict';
+
+const { generateDigits } = require('./generateDigits');
+const { terminal } = require('./terminal');
+const conceivedDigits = generateDigits();
+
+function askQueston() {
+  let bulls = 0;
+  let cows = 0;
+
+  terminal.question('Guess the number ', (guessedNumber) => {
+    const guessedDigits = guessedNumber.split('').map(Number);
```

Comment:
I think it would be great to write down the rules of the game here. Something like "You need to guess a number with 4 digits